### PR TITLE
Ported fix onto master to show that it doesn't work

### PR DESCRIPTION
**Pull request by GrahamCampbell**
Demonstrates that https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2851 doesn't work on master.

### CODE REVIEW

**Comment by SpacePossum** [comment]
thanks for checking, looking at it I think this might a issue with removing cleared tokens before testing the tokens match the expected tokens.

File: tests/Tokenizer/TokensTest.php [hunk 1]
```diff
@@ -888,6 +888,27 @@ public function provideEnsureWhitespaceAtIndexCases()
     {
         return [
             [
+                '<?php echo 1;',
```
Comment (SpacePossum) [inline tests/Tokenizer/TokensTest.php]: Please add this at line 883
```php
        $this->assertTrue($tokens->isChanged(), 'Tokens collection must be marked as changed after fixing.');
        $tokens->clearEmptyTokens();
```
after that the tests should be green

Comment (GrahamCampbell) [inline tests/Tokenizer/TokensTest.php]: Done.

Comment (SpacePossum) [inline tests/Tokenizer/TokensTest.php]: thanks :)
>$this->assertTrue($tokens->isChanged()

this was incorrect of me, as not all tests make a change, sorry about that :(

**Comment by SpacePossum** [comment]
replaced by https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2870

**Pull request closed without merging**